### DISCUSSION TITLE
Meeting: Celo Wallet and (d)App Council #12 [June] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#9625] [Wallet and (d)App Council]
# Monthly Celo Wallet and (d)App Council

##  📆 Quick links 

- When: **Wednesday, June 22 (9am PDT/6pm CEST)**
- Where: **[Google Meet](https://meet.google.com/uwj-wfse-uzm)**
- Agenda: **comments**👇
- Previous: https://github.com/celo-org/celo-monorepo/discussions/9524
- Recording: [Google Drive](https://drive.google.com/file/d/1QulkS13DeBdT4b8i41L1e8GQYaFsUjUv/view?usp=sharing)

## 🤷‍♂️ What is this?

A monthly meeting for Celo **developers, founders,** **product** people (anyone really!) to come together and
- find consensus (coordinate),
- learn about needs, pain points, and desires
- present and collect technical feedback

## 🗒️  Agenda

- Comment with a short sentence describing what you'd like to discuss
- Don't worry, it can be small or big! 
- Every builder is welcome (and we are very easy going, I promise 😄)
- Upvote topics you're interested in

## Housekeeping

_You can safely ignore this!_

- [x] Send update calendar invite with Github link
- [x] Encourage additions to the agenda
- [x] Publicise on forum and Discord
- [ ] Appoint secretary (to take notes in Github comment section ✍️)
- [x] RECORD meeting 🎥

**Comment by aaronmgdr** [comment]
react-celo 4.0.1 Beta is out.

**Reply by arthurgousset** [reply]
Meeting notes: 

- small css fix and hiding default wallets
- decided not to rush a 4.1.0 update to give it time to be stable (will probably happen next month) 

@jeanregisser:

- Question: what's the plan for popular dapps to integrate react-celo? Is anyone chasing them?

@aaronmgdr: 

- Loosely me

Eric:

- Dylon and Kyle found that the default provider is making the experience too slow.

**Reply by aaronmgdr** [reply]
my discord `aaron | clabs #8858`

**Comment by ole-nov** [comment]
Call for action: protecting end-users from high slippage when transacting with Mento
https://www.notion.so/clabsco/Proposal-Mento-Consumer-protection-from-high-slippage-76e14c0cd8014dcbbb9a1951ce8ddfee

**Reply by arthurgousset** [reply]
Meeting notes:

- there is a problem with high slippage on Mento (though it depends on the pairs)
- in Mento slippage exists to protect the reserve from arbitrageurs
- this is good for Mento, but bad for end-users
- Mento was not meant for use by end-users in the first place, swapping was meant to take place on DEXs
- Call to action:
  - recommendation that wallets and DEXs use [swappa](https://github.com/terminal-fi/swappa) for swapping instead of Mento
  - recommendation that frontend providers inform their users about slippage better

**Reply by arthurgousset** [reply]
Eric:

- Instead of bucket sizes, have you considered charging a flat fee for bundled trades (i.e. "hops where you route swaps over different pairs").
- Are you considering lowering the mint and burn rate to be more competitive
- Current version of swappa can be even faster, Dylon has a faster version (reach out to us if you're interested)

**Reply by arthurgousset** [reply]
Evan:

- we're decentralising our Ubeswap frontend and providing a referral fee (e.g. Gonza's celotracker could provide a swapping frontend)

@mattclabs:

- Are you considering becoming an aggregator link 1inch? 

Evan:

- No not really at the moment.

**Reply by arthurgousset** [reply]
@mattclabs: 

- How do you monetise swaps that you route through Mobius?

Evan:

- You want to monetise the frontend so $UBE (DAO) token holders benefit from the popularity of the frontend.

**Reply by arthurgousset** [reply]
@patricknbaron: 

- Has there been any discussion around increasing cREAL liquidity which is quite thin at the moment?

@aleksey-nov: 

- Yes, it's on our mind, we're probably increasing bucket sizes by 4x.

@mattclabs: 

- Have we revised our cSTABLES strategy since competing stables are coming in?

@aleksey-nov: 

- Currently focusing on keeping existing tokens stable. But we should think about launching more carefully going forward.

**Comment by product-matt** [comment]
launch of stCELO liquid staking derivative on CELO in July
https://github.com/celo-org/staked-celo

**Reply by arthurgousset** [reply]
Meeting notes:

- "Lido or Rocket Pool of Celo"
- going for a non-rebasing approach (more Rocket Pool than Lido)
- reach out to @mattclabs if you are interested in using or supporting stCELO
- Outstanding questions:
  1. upgradability of smart contract will depend on a multisig (this is an MVP)
    - at the moment, we choose people on the team
    - but the main security comes from a **3 day time lock**, which will be visible on a web page.
  2. validator election using stCELO tokens
    - constraints: maximise APR for stCELO holders while being neutral (ideally no influence on elected validators)
    - we want to outline the election process systematically so our neutrality is verifiable 

Contact @mattclabs on Discord: `matt#9000`

**Reply by arthurgousset** [reply]
@jeanregisser: 

- I maintain [churrito.fi](https://churrito.fi/). How should I support stCELO there?
- I'm very excited about this!

**Reply by arthurgousset** [reply]
Evan: 

- How does this relate to the EGG initiative?

@mattclabs: 

- We changed our prioritise re EGG for now. But stCELO as a liquid staking derivative is an important building block for any feature yield protocols.

**Reply by arthurgousset** [reply]
Eric:

- what about the crosschain version of EGG on abacus?

@mattclabs:

- That would have been an even more long-term vision. So not part of main prioritise at the moment.
- Goal is still to bring yield to Celo, but it's still in the early product phase. 
- We all realised, the bear market changes yield parameters a lot.
- My focus is to find sources of yield beyond web3.

Evan: 

- Link: https://celo.symm.fi/#/pool/0xa287a3722c367849efa5c76e96be36efd65c290e000100000000000000000020
- you can mint cDEFI token on symmetric and diversify your token holdings
- the pool rebalances the holdings (so it serves as an index)
- ![image](https://user-images.githubusercontent.com/46296830/175092396-d56fdd53-d88e-43e4-8792-64e9d4f1b674.png)

@jeanregisser:

- Is anyone building leverage using stCELO?
- Similar project: https://indexcoop.com/interest-compounding-eth-index

@patricknbaron:

- we anticipate supporting stCELO in relatively short period of time (Q3/Q4)
- we also have contracts deployed for 5x leverage (but no frontend support yet)

**Comment by arthurgousset** [comment]
Robin: Question re Valora support in countries and adoption

@jeanregisser:

- there are some restrictions around sanctioned countries (but it's a limited list)
- phone number verification system works better in some countries than others

Robin: Is there another mobile wallet?

- Node.fi (https://nodefinance.org/) is a B2B play, so we ship an SDK so apps can create wallets

**Comment by fbaud** [comment]
Looking forward to meeting some of you guys at the Celo Connect Salon Paris - 17-22 July https://celoconnect.com/ethcc2022/

If for some reasons, we can not meet there, we could have a drink together in Paris during that week. You can contact me at fbaud@primusmoney.fr or through discord at `Frederic#0038`

**Reply by arthurgousset** [reply]
Thanks for sharing @fbaud! See you in Paris 🇫🇷

**Comment by arthurgousset** [comment]
📽 Here is the [recording](https://drive.google.com/file/d/1QulkS13DeBdT4b8i41L1e8GQYaFsUjUv/view?usp=sharing) for anyone that missed the meeting.
